src/components/Dropdown/__tests__/__snapshots__/dropdown-snapshot-tests.jest.js.snap

```diff
@@ -1893,6 +1893,7 @@ exports[`Dropdown renders correctly with leftAvatar 1`] = `
               Ocean
             </div>
             <button
+              aria-busy={false}
```

Comment:
Is this change legit? I've seen it pop all over snapshot files.

Reply:
Yes I checked it before and it's legit in terms of a11y